<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>421</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>768</width>
    <height>421</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="18" rowspan="3" alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="startserver">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>开启服务器</string>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icon/recourse/start.png</normaloff>:/icon/recourse/start.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>48</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="processbutton">
      <property name="minimumSize">
       <size>
        <width>72</width>
        <height>72</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>82</width>
        <height>75</height>
       </size>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;显示进程信息&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icon/recourse/process.png</normaloff>:/icon/recourse/process.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>72</width>
        <height>72</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QPushButton" name="exitbutton">
      <property name="minimumSize">
       <size>
        <width>72</width>
        <height>72</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>82</width>
        <height>75</height>
       </size>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;退出程序&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icon/recourse/exit.png</normaloff>:/icon/recourse/exit.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>72</width>
        <height>72</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="filebutton">
      <property name="minimumSize">
       <size>
        <width>72</width>
        <height>72</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>82</width>
        <height>75</height>
       </size>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;显示文件信息&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icon/recourse/file.png</normaloff>:/icon/recourse/file.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>72</width>
        <height>72</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="screenbutton">
      <property name="minimumSize">
       <size>
        <width>72</width>
        <height>72</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>82</width>
        <height>75</height>
       </size>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;监控屏幕&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icon/recourse/screen.png</normaloff>:/icon/recourse/screen.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>72</width>
        <height>72</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="buiderbutton">
      <property name="minimumSize">
       <size>
        <width>72</width>
        <height>72</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>82</width>
        <height>75</height>
       </size>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;生成客户端&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icon/recourse/builder.png</normaloff>:/icon/recourse/builder.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>72</width>
        <height>72</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="aboutbutton">
      <property name="minimumSize">
       <size>
        <width>72</width>
        <height>72</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>82</width>
        <height>75</height>
       </size>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;关于本程序&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icon/recourse/about.png</normaloff>:/icon/recourse/about.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>72</width>
        <height>72</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="18">
     <widget class="QLabel" name="addlabel">
      <property name="text">
       <string>服务器所有IP(IPV4):</string>
      </property>
     </widget>
    </item>
    <item row="1" column="18">
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item row="5" column="18">
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0" rowspan="10" colspan="10">
     <widget class="QTableWidget" name="tableWidget">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>300</height>
       </size>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>6</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </item>
    <item row="4" column="18">
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="inputMask">
       <string>00000</string>
      </property>
      <property name="text">
       <string>10086</string>
      </property>
      <property name="maxLength">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item row="10" column="18">
     <widget class="QLabel" name="statslabel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Server not Start</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="11" column="18">
     <widget class="QLabel" name="timelabel">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="18">
     <widget class="QLabel" name="portlabel">
      <property name="text">
       <string>端口:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="18" alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="stopserver">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>停止服务器</string>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icon/recourse/stop.png</normaloff>:/icon/recourse/stop.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>48</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="cmdbutton">
      <property name="minimumSize">
       <size>
        <width>72</width>
        <height>72</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>82</width>
        <height>75</height>
       </size>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;发送CMD命令&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icon/recourse/cmd.png</normaloff>:/icon/recourse/cmd.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>72</width>
        <height>72</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>exitbutton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>558</x>
     <y>478</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_GetData()</slot>
 </slots>
</ui>
